src/Uno.Toolkit.UI/Controls/ZoomContentControl/ZoomContentControl.cs

```diff
@@ -0,0 +1,782 @@
+
+	public static readonly DependencyProperty VerticalZoomCenterProperty =
+	DependencyProperty.Register(nameof(VerticalZoomCenter), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	public static readonly DependencyProperty ScaleWheelRatioProperty =
+	DependencyProperty.Register(nameof(ScaleWheelRatio), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0006d));
+
+	public static readonly DependencyProperty PanWheelRatioProperty =
+	DependencyProperty.Register(nameof(PanWheelRatio), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.25d));
+
+	public static readonly DependencyProperty IsPanAllowedProperty =
+	DependencyProperty.Register(nameof(IsPanAllowed), typeof(bool), typeof(ZoomContentControl), new PropertyMetadata(defaultValue: true));
+
+	public static readonly DependencyProperty HorizontalOffsetProperty =
+	DependencyProperty.Register(nameof(HorizontalOffset), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	public static readonly DependencyProperty VerticalOffsetProperty =
+	DependencyProperty.Register(nameof(VerticalOffset), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	public static readonly DependencyProperty VerticalMaxScrollProperty =
+	DependencyProperty.Register(nameof(VerticalMaxScroll), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	public static readonly DependencyProperty VerticalMinScrollProperty =
+	DependencyProperty.Register(nameof(VerticalMinScroll), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	public static readonly DependencyProperty HorizontalMaxScrollProperty =
+	DependencyProperty.Register(nameof(HorizontalMaxScroll), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	public static readonly DependencyProperty HorizontalMinScrollProperty =
+	DependencyProperty.Register(nameof(HorizontalMinScroll), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	public static readonly DependencyProperty HorizontalScrollValueProperty =
+	DependencyProperty.Register(nameof(HorizontalScrollValue), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	public static readonly DependencyProperty VerticalScrollValueProperty =
+	DependencyProperty.Register(nameof(VerticalScrollValue), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	public static readonly DependencyProperty ViewPortWidthProperty =
+	DependencyProperty.Register(nameof(ViewPortWidth), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	public static readonly DependencyProperty ViewPortHeightProperty =
+	DependencyProperty.Register(nameof(ViewPortHeight), typeof(double), typeof(ZoomContentControl), new PropertyMetadata(0.0d));
+
+	//Not a real margin, but a spacing to avoid overlapped by ShellControlPanel
+	public Thickness AdditionalMargin
```

Comment:
consider naming this to ContentPadding, or just leveraging the existing ContentControl.Padding

Reply:
also this should be in the doc